extensions/kill-process/package.json

```diff
@@ -73,9 +80,27 @@
       "label": "Show Process Path",
       "name": "shouldShowPath",
       "required": false,
+      "default": false,
       "title": "",
       "type": "checkbox"
     },
+    {
+      "description": "Close window after killing a process",
+      "label": "Close window after kill",
+      "name": "closeWindowAfterKill",
+      "required": false,
+      "default": true,
+      "title": "After Killing",
```

Comment:
Since we have this as a title (heading), I think we can label the two actions "Close window" and "Clear search bar". @pernielsentikaer what do you think?

Reply:
I agree, as it's pretty redundant